package.json

```diff
@@ -51,7 +51,7 @@
     "minimist": "^1.2.0",
     "npmlog": "^4.1.2",
     "plop": "^2.0.0",
-    "prettier": "^1.14.2",
+    "prettier": "^1.16.1",
```

Comment:
This upgrade is necessary to enable the `endOfLine` config option. (See https://prettier.io/docs/en/options.html#end-of-line)

Reply:
if upgrading prettier...please run `yarn prettier` to update all formatting across the project w/ this PR. It seems we also need some updates now in `.prettierignore` now that i try this locally, but i think it can be done in a different PR 😄

Reply:
with the conversion to typescript/tslint for pf4 upcoming, i don't know how much time we invest on those eslint rules for pf4 - @dgutride can probably weigh this. Regarding prettier though - i think we should just ensure running `yarn prettier` is consistent for now (certainly realize the other issue exists). I think the pf4 team will have to weigh this. Running `yarn prettier` to format our code now is low effort though ;)

Reply:
f23d5ca44f83db8dcd3d1103ec2a7a885107b48a fixes the formatting for the pf3 stuff. The build is still broken, but less badly now I suppose, trying to figure out what else is wrong with it...

The linting is not failing anymore at least. If you want me to any other formatting, I'm happy to!